build/release/teamcity-support.sh

```diff
@@ -15,3 +15,33 @@ tc_start_block() {
 tc_end_block() {
   echo "##teamcity[blockClosed name='$1']"
 }
+
+log_into_gcloud() {
+  # Set up Google credentials.
+  if [[ "${GOOGLE_EPHEMERAL_CREDENTIALS}" ]]; then
+    echo "${GOOGLE_EPHEMERAL_CREDENTIALS}" > creds.json
```

Comment:
maybe put this in a different directory than the cwd, i think some CI checks that the current directory has no changes.

Reply:
Hrm. I was following the pattern from the `build/teamcity-nightly-roachtest.sh` script. Are the CI checks you're thinking about the steps to build cockroach? If so , then how about making this file is get created _after_ the cockroach builds are completed?

Reply:
I moved the file to `~` and delete when the script finishes running.